haiku-apps/bescreencapture/bescreencapture-2.8.0.recipe

```diff
@@ -5,36 +5,25 @@ to any media format supported in Haiku.
 BeScreenCapture can record either the entire screen, or just a section you \
 select."
 HOMEPAGE="https://github.com/jackburton79/bescreencapture/releases"
-COPYRIGHT="2014-2022 Stefano Ceccherini"
+COPYRIGHT="2014-2023 Stefano Ceccherini"
 LICENSE="BSD (3-clause)
 	MIT"
 REVISION="1"
 SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="b701e4b322dbbca449945c74c5e088f04d5757b9c98297bb1b62cc41c0ac0aa7"
+CHECKSUM_SHA256="d5fb33eec0b5d57e5b5c1a0ae854d7eaa3dade85bda1a7ea395a8565792648d0"
 SOURCE_FILENAME="bescreencapture-$portVersion.tar.gz"
-
+SOURCE_DIR="bescreencapture-$portVersion"
 ARCHITECTURES="all"
 
 PROVIDES="
 	bescreencapture = $portVersion
 	app:BeScreenCapture = $portVersion
-	"
-REQUIRES="
-	haiku
-	ffmpeg_tools
-	"
-
-SUMMARY_inputfilter="Shortcut handler for BeScreenCapture"
-DESCRIPTION_inputfilter="Input Server Addon for BeScreenCapture. Allows the \
-user to launch BeScreenCapture and start/stop recording using a keyboard \
-combination (CTRL-COMMAND-SHIFT + R)."
-PROVIDES_inputfilter="
 	bescreencapture_inputfilter = $portVersion
 	app:BeScreenCaptureInputFilter = $portVersion
 	"
-REQUIRES_inputfilter="
-	bescreencapture == $portVersion base
+REQUIRES="
 	haiku
+	ffmpeg_tools
```

Comment:
This will use ffmpeg-4.2.9 and not the newer ffmpeg5 (or upcomming ffmpeg5), maybe switch to `cmd:ffmpeg` here?

Reply:
Ran a check with ffmp5_tools installed, build/installed and then installed ffmpeg6_tools, worked fine, if I do this with ffmpeg_tools in the recipe it will conflict with other versions from ffmpeg, cmd:ffmpeg should fix this. :)